pkgs/development/python-modules/click-aliases/default.nix

```diff
@@ -30,6 +27,8 @@ buildPythonPackage rec {
 
   pythonImportsCheck = [ "click_aliases" ];
 
+  passthru.updateScript = nix-update-script { };
+
```

Comment:
This is not needed. `nix-update` works usually fine on Python modules.

```suggestion

```

Reply:
I just tried `nix-shell ./maintainers/scripts/update.nix --argstr package python3Packages.click-aliases` on `master` and it did not work.

Reply:
I removed the updateScript as requested, even though updating the package then doesn't work automatically.